RecycleView/app/src/main/java/net/boondockradio/recycleview/ItemViewHolder.java

```diff
@@ -0,0 +1,31 @@
+package net.boondockradio.recycleview;
+
+import android.content.Context;
+import android.support.v7.widget.RecyclerView;
+import android.view.View;
+import android.widget.TextView;
+import android.widget.Toast;
+
+public class ItemViewHolder extends RecyclerView.ViewHolder {
```

Comment:
like this

https://github.com/bowyer-app/PlayerManage/blob/develop/app/src/main/java/com/bowyer/app/playermanage/ui/adapter/PlayerAdapter.java#L73-L91